ports/libiconv/portfile.cmake

```
@@ -1,6 +1,6 @@
 if(NOT VCPKG_TARGET_IS_WINDOWS)
     set(VCPKG_POLICY_EMPTY_PACKAGE enabled)
-    file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/share/Iconv)
+    file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/share/${PORT})
```

```suggestion
    file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/share/iconv)
```

the mkdir is for the copy below. The port is named libiconv